pkg/cmd/uninstall/uninstall.go

```diff
@@ -236,6 +236,13 @@ func (o *UninstallOptions) deleteNamespace(namespace string) error {
 // of an unrelated error
 func (o *UninstallOptions) DeleteReleaseIfPresent(ns string, releaseName string, errors []error, force bool) []error {
 	if err := o.Helm().StatusRelease(ns, releaseName); err == nil || force {
+
+		err = o.setPVCFinalizerToNull(ns, releaseName)
+
+		if err != nil {
+			errors = append(errors, fmt.Errorf("Failed to set PVC finalizer to null in %s helm chart in namespace %s: %s", releaseName, ns, err))
```

Comment:
Any reason we don't return when we see an error? curious ...
More specifically, should we even go to the next step (in this case DeleteRelease) if an error is encountered?

Reply:
Good point, let me fix that and return immediately!

Reply:
Fixed!